content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,47 @@ menu:
 weight: 101
 ---
 
+## v2.0.8 [2021-08-09]
+
+### Go version
+
+- Upgrade to Go 1.16. **Requires macOS Sierra 10.12 or later to run.**
+
+### Linux packaging for influx CLI
+
+- Modify Linux packages (`.deb` and `.rpm`) to enable standalone installation of the `influx` CLI for InfluxDB Cloud users, includes the following changes:
+
+  - Add new `influxdb2-cli` package that contains only the `influx` CLI binary.
+  - Remove the `influx` CLI binary from the `influxdb2` package. Now, the `influxdb2` package declares a recommended dependency on the `influxdb2-cli` package.
```

Comment:
Are the go version upgrade and linux re-packaging considered breaking changes?

Reply:
To be sure, will users need to take any action during installation/upgrade to re-install the influx CLI, or will the new "recommended" dependency automatically install w/o action?

Reply:
Go version should have no breaking impact other than bumping the minimum macOS version.

Packaging could be a breaking change for users, depending on how they install.
* Users who install via `apt install` or `apt upgrade` shouldn't see a breaking change because `apt` will handle fetching & installing the recommended dependency
* Users who manually `wget` the deb and `dpkg -i` will see a breaking change; they'll need to download and install both `influxdb2-2.0.8-amd64.deb` and `influxdb2-client-2.0.8-amd64.deb` to get the CLI

Same for the RPM equivalent of the above commands.